docs/pipeline.rst

```diff
@@ -71,3 +84,9 @@ DataNode
 --------
 .. autoclass:: DataNode
    :members:
+
+
+Decorator
```

Comment:
This subtitle is subpar.

Reply:
Done

Reply:
Suits me good.

Reply:
Not good at all.
Something like `Pipeline factor decorator` would be more appropriate.

Reply:
```suggestion
Pipeline factory decorator
```
or 
```suggestion
Pipeline definition decorator
```
would be better ideas.